MapSample/MapSample/Platforms/Windows/App.xaml.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.UI.Xaml;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace MapSample.WinUI;
+
+/// <summary>
+/// Provides application-specific behavior to supplement the default Application class.
```

Comment:
/// Provide application-specific behavior to supplement the default Application class.